NitroxModel/DataStructures/GameLogic/Entities/Metadata/PrecursorKeyTerminalMetadata.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using ProtoBufNet;
+
+namespace NitroxModel.DataStructures.GameLogic.Entities.Metadata
+{
+    [Serializable]
+    [ProtoContract]
+    public class PrecursorKeyTerminalMetadata : EntityMetadata
+    {
+        [ProtoMember(1)]
+        public bool Slotted { get; }
+
+        public PrecursorKeyTerminalMetadata()
```

Comment:
This does not fit the implementation of other Metadata classes which can result in json saving problems. Pls look at other (up to date) Metadata classes.

Reply:
By this are you refering to the protected decleration found on other metadata classes?

Reply:
Yes 😉